### DISCUSSION TITLE
vook_web_v3のデプロイ方法（20230423）

### DISCUSSION THREAD

**Discussion by atsushimemet** [atsushimemet/vook_web_v3#24] [General]
Ref:
- https://github.com/atsushimemet/vook_web_v3/issues/5

**Comment by atsushimemet** [comment]
上記一部修正しながらデプロイ作業を実施。

**Comment by garammasala29** [comment]
httpのみ、https未対応の場合
1. ドメインのネームサーバーをRoute53に変更
    - 既存のホストゾーンは削除しておく
    - 本番アカウントでRoute53でホストゾーンを作成
    ドメイン名とパブリックホストゾーンを選択し作成。ネームサーバー（NS）を確認
    - NSをお名前.comからRoute53に変更
    （すでにセットされていると思うので書き換える。多分4つ。反映まで時間がかかる）
2. ドメインに紐付くIPアドレスを登録
    - Route53でレコードセットを作成
  値にElasticIPを入力しAレコードを登録
   <img width="709" alt="image" src="https://user-images.githubusercontent.com/69446373/233832664-b71ccfbb-1eee-482e-b8fd-c57f51ec0a86.png">
3. 反映されたか確認
`dig vook.tokyo NS +short`で設定したNSが表示されていればOK、URLにアクセスできる

ここで思い出したけど、確かにこのあたりでnginx起動してないなとあった気がするので調査しておきます。
